WordPress/src/main/java/org/wordpress/android/util/config/UnifiedAboutFeatureConfig.kt

```diff
@@ -1,19 +0,0 @@
-package org.wordpress.android.util.config
-
-import org.wordpress.android.BuildConfig
-import org.wordpress.android.annotation.Feature
-import org.wordpress.android.util.config.UnifiedAboutFeatureConfig.Companion.UNIFIED_ABOUT_REMOTE_FIELD
-import javax.inject.Inject
-
-@Feature(UNIFIED_ABOUT_REMOTE_FIELD, true)
-class UnifiedAboutFeatureConfig @Inject constructor(
-    appConfig: AppConfig
-) : FeatureConfig(
-        appConfig,
-        BuildConfig.UNIFIED_ABOUT,
```

Comment:
We can also remove `UNIFIED_ABOUT` from `build.gradle`.

Reply:
Yes, 👍